extensions/wikia/CommunityPage/styles/components/_header.scss

```diff
@@ -45,22 +48,47 @@ $hero-image-background: rgba(0,0,0,0.35);
 		font-size: 36px;
 		font-weight: bold;
 		line-height: 48px;
+		margin:	0;
+	}
+
+	h3.community-page-subheading {
+		color: $color-white;
+		font-size: 24px;
```

Comment:
please include: `@import 'extensions/wikia/DesignSystem/bower_components/design-system/base/typography';` and use font-size & `line-height: $lihe-height;` from there